spec/features/operator/clusterConfig/clusterConfig_test.go

```diff
@@ -0,0 +1,29 @@
+package clusterConfig
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestClusterConfigReinstall(t *testing.T) {
+	Convey("Given the Operator is installed", t, func() {
+		Convey("Delete cluster config from cluster", nil)
```

Comment:
before testing effects deleting config, shouldn't we test installing operator installed pipelines, triggers and cluster tasks

Reply:
```
Convey("Given the Operator is installed", t, func() {
    Convey("It should install the following", func() {
        Convey("installs Pipelines 0.8", nil)
	Convey("installs Triggers 0.1", nil)
	Convey("It should install the following", func() {
        ...
        }
    Convey("Delete cluster config from cluster", nil)
    Convey("Create new config name 'Cluster' with target namespace 'openshift-pipelines'", nil)
    Convey("It should install the following", func() {
        Convey("installs Pipelines 0.8", nil)
	Convey("installs Triggers 0.1", nil)
	Convey("It should install the following", func() {
        ...
        }
```

Reply:
Sure, But there is one more test which Does this https://github.com/praveen4g0/release-tests/blob/master/spec/install/olm/install_test.go#L9, I hope this part can be skipped here, it leads to duplication any how after Reinstall We need to check those

Reply:
i see :+1: